src/screens/SurveyOverviewScreen.tsx

```diff
@@ -0,0 +1,83 @@
+import { useNavigation } from '@react-navigation/core';
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { useCallback } from 'react';
+import { useQuery } from 'react-apollo';
+import { SectionList } from 'react-native';
+
+import { SectionHeader, TextListItem } from '../components';
+import { LoadingSpinner } from '../components/LoadingSpinner';
+import { texts } from '../config';
+import { combineLanguages } from '../helpers';
+import { usePullToRefetch, useSurveyLanguages } from '../hooks';
+import { SURVEYS } from '../queries/survey';
+import { Survey } from '../types';
+
+const useSurveySections = () => {
+  const { data: surveys, loading, refetch } = useQuery<{
+    ongoing: Survey[];
+    archived: Survey[];
+  }>(SURVEYS, { fetchPolicy: 'cache-and-network' });
+
+  const sectionData = [];
```

Comment:
rename to `surveySections` to be able to return `{ loading, refetch, surveySections };`? or is `sectionData` somehow better for readability?

Reply:
hmm. can't think of a good reason to not rename it, so i will do it

Reply:
done